lib/laa_crime_schemas/structs/offence.rb

```diff
@@ -4,7 +4,7 @@ module LaaCrimeSchemas
   module Structs
     class Offence < Base
       attribute :name, Types::String
-      attribute? :offence_class, Types::String.optional
+      attribute? :offence_class, Types::OffenceClass.optional
```

Comment:
Until we don't do the splitting of the offences, if we introduce this change, it will break many existing datastore applications (as in, not possible to read them back) as many have multi-class in this attribute (i.e.`F/G/K`) and the type will forbid that.

Reply:
Aha, good catch! Thank you.

Reply:
In reality once we do the splitting we will have to wipe out the datastore staging database and start fresh. It will just be not possible to map old multi class offences to new split ones. The offence name will change quite dramatically in some scenarios, not possible or too much effort to create a data migration.